rotkehlchen/globaldb/migrations/migration2.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from typing import TYPE_CHECKING
+
+from rotkehlchen.constants.timing import ETH_PROTOCOLS_CACHE_REFRESH
+from rotkehlchen.globaldb.cache import (
+    globaldb_get_general_cache_values,
+    globaldb_set_general_cache_values_at_ts,
+)
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import GeneralCacheType, Timestamp
+from rotkehlchen.utils.misc import ts_now
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBConnection
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def globaldb_data_migration_2(conn: 'DBConnection') -> None:
+    """Introduced at 1.28.0
+    - Resets curve cache last queried timestamp to query and save gauges
+    """
+    with conn.write_ctx() as write_cursor:
```

Comment:
My bad @nebolax. I mentioned migration, but it's not needed. There is already a DB upgrade scheduled for global DB 1.28. Unless I am mistaken v4_v5 global DB upgrade comes at 1.28. Which means we do not need a migration if there is already a DB ugprade. Just add this code as part of the global DB upgrade and test it there.

No need to have a DB and a migration in the same release (unless migration was a long running process).